src/Resources/skeleton/Story.tpl.php

```diff
@@ -0,0 +1,12 @@
+import <?php echo $componentName; ?> from "../templates/<?php echo $template; ?>";
```

Comment:
Relates to the previous comment on stories path: I'd prefer the full path to be dynamically generated depending on where the story file is created.